backend/src/Controller/ChatRoom/ChatRoomController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Controller\ChatRoom;
+
+use App\Controller\BaseController;
+use App\Service\ChatRoom\ChatRoomService;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Serializer\SerializerInterface;
+use OpenApi\Annotations as OA;
+use Nelmio\ApiDocBundle\Annotation\Security;
+
+/**
+ * fetch chat room.
+ * @Route("v1/chatroom/")
+ */
+class ChatRoomController extends BaseController
+{
+    private $chatRoomService;
```

Comment:
parameter type is required in parameter definition.

Reply:
It doesn't make sense to repeat the variable definition once in the comments and once within the builder, but I'll do it